crates/flox/src/commands/environment.rs

```diff
@@ -452,14 +452,29 @@ impl Uninstall {
         let mut environment =
             resolve_environment(&flox, self.environment.as_deref(), "uninstall").await?;
 
-        environment
+        let packages_str = packages_to_string(&packages);
+        let plural = packages.len() > 1;
+
+        if environment
             .uninstall(
                 packages.drain(..),
                 &flox.nix(Default::default()),
                 &flox.system,
             )
             .await
-            .context("could not uninstall packages")?;
+            .context("could not uninstall packages")?
+        {
+            println!(
+                "🗑️ Uninstalled {packages_str} from '{}' environment.",
```

Comment:
Just to keep in mind, emojies may/will be unsupported on some platforms / terminals.
Eventually we want a way to filter these strings when emojis are unsupported.

Not a change for today!

Reply:
Good to be thinking about